### PR TITLE
Moved Credit Market report link to Balance. Added feedback card

### PR DESCRIPTION
**Pull request by amichard**
#1568 

Changelog:
- Changed Requested Supplemental to Supplemental Requested
- Added Feedback card
- Minor text changes

### CODE REVIEW

**Comment by amichard** [comment]
[Dashboard | Enhancements to the dashboard |2|](https://trello.com/c/mFxUVRP5/2065-dashboard-enhancements-to-the-dashboard-2)

File: frontend/src/constants/notificationTypes.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const NOTIFICATION_TYPES = {
   COMPLIANCE_REPORT_RECOMMENDED_FOR_REJECTION_MANAGER: 'Compliance Report Recommended for Rejection - Manager',
   COMPLIANCE_REPORT_ACCEPTED: 'Compliance Report Accepted',
   COMPLIANCE_REPORT_REJECTED: 'Compliance Report Rejected',
-  COMPLIANCE_REPORT_REQUESTED_SUPPLEMENTAL: 'Compliance Report Requested Supplemental',
+  COMPLIANCE_REPORT_REQUESTED_SUPPLEMENTAL: 'Compliance Report Supplemental Requested',
```
Comment (plasticviking) [inline frontend/src/constants/notificationTypes.js]: This wording seems awkward to me. intentional?

File: frontend/src/constants/notificationTypes.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ const NOTIFICATION_TYPES = {
   EXCLUSION_REPORT_RECOMMENDED_FOR_REJECTION_MANAGER: 'Exclusion Report Recommended for Rejection - Manager',
   EXCLUSION_REPORT_ACCEPTED: 'Exclusion Report Accepted',
   EXCLUSION_REPORT_REJECTED: 'Exclusion Report Rejected',
-  EXCLUSION_REPORT_REQUESTED_SUPPLEMENTAL: 'Exclusion Report Requested Supplemental'
+  EXCLUSION_REPORT_REQUESTED_SUPPLEMENTAL: 'Exclusion Report Supplemental Requested'
```
Comment (plasticviking) [inline frontend/src/constants/notificationTypes.js]: Also seems awkward

File: frontend/src/constants/notificationTypes.js (hunk 1 above)
Comment (amichard) [reply frontend/src/constants/notificationTypes.js]: Ya, it is intentional. I just copied the suggestion from trello.

File: frontend/src/constants/notificationTypes.js (hunk 2 above)
Comment (amichard) [reply frontend/src/constants/notificationTypes.js]: Ya, it is intentional. I just copied the suggestion from trello.

**Pull request merged**